api/models/artist.py

```diff
@@ -13,15 +13,18 @@ class Artist(Base):
     user_id = mapped_column(ForeignKey("users.id"))
     user = relationship("User", back_populates="artist", lazy="joined")
     payment_urls = relationship("PaymentUrl", lazy="joined")
+    social_links = relationship("SocialLink", lazy="joined")
     media = relationship("Media")
```

Comment:
Should this also be lazy loaded?  I feel like media is the most likely speedbump in our queries, given how big the files could be, right?